### ISSUE TITLE
Running twice results in malformed credentials file

### ISSUE THREAD

**Issue by stevenjohnstone** [cevoaustralia/aws-google-auth#22]
On Ubuntu with version 0.0.12. Two runs of aws-google-auth results in  ~/.aws/credentials with two "[None]" sections. The config parser of aws-cli throws error configparser.DuplicateSectionError.

**Comment by rschurchill** [comment]
On Windows I get the credentials duplicated in the cred and config files on multiple runs but no duplicate section error.  For me the [None] sections no longer were added when i specified a profile using the -p switch, with that the intended profile was overwritten correctly each time.

**Comment by stevemac007** [comment]
Yep - confirm I get the erronious behaviour on OSX too - It looks like it tries to write the profile and credentials section in a confusing way when the `-p` flag is not supplied.

Will look into a fix shortly.

**Comment by mide** [comment]
I found the cause of this. In `__init__.py`

```python
print("Credentials Expiration: " + format(token['Credentials']['Expiration'].astimezone(get_localzone())))
if config.profile is None:
    print_exports(token)

_store(config, token)
```

It looks like regardless of the `config.profile` being set or not, we're always storing the credentials. By adding an `else`, we will print the `export` command when it's not set, and save to the AWS config / credentials when it is set. 

```python
print("Credentials Expiration: " + format(token['Credentials']['Expiration'].astimezone(get_localzone())))
if config.profile is None:
    print_exports(token)
else:
    _store(config, token)
```

I am working on testing this change and if all goes well, I will have a PR shortly.

**Comment by mide** [comment]
I've opened up #32 to address this issue.

**Comment by stevemac007** [comment]
Agree, that seems to be the issue - I've been working on trying to get some tests around the cli file to validate these scenarios.

**Comment by stevemac007** [comment]
As per discussion in #32 - we need to decide what to do if no profile is supplied.

Initially I thought if it wasn't supplied, just print the creds and leave the config alone, but the suggestion from @tjbaker was to use `default`, I'm actually thinking we should use the current `AWS_PROFILE` environment variable.

If you are running this in a shell where your `AWS_PROFILE=dev`, you probably expect that profile to get renewed..

Thoughts?

**Comment by tjbaker** [comment]
I memory failed me.  The previous library we used would default to `sts` to the profile was unspecified.  This is safer.  A user may have IAM access keys in the `default` profile waht we wouldn't want to clobber.

See here:
https://github.com/ThoughtWorksInc/aws_role_credentials/blob/master/README.rst#options

**Comment by mide** [comment]
I have opened PR #34 to switch to a default of `sts`.

**Comment by stevemac007** [comment]
Thanks @mide - these changes have been released as part of 0.0.15.
